src/main/resources/i18n-jfrog.properties

```diff
@@ -312,6 +312,15 @@ artifactory.task.publishBuildInfo.header.username.description = Override the def
 artifactory.task.publishbuildinfo.header.password = Override Default Password
 artifactory.task.publishbuildinfo.header.password.description = The password of the user entered above.
 
+#Artifactory Collect Issues
+artifactory.task.collectIssues.title = Artifactory Collect Issues
+artifactory.task.collectIssues.configFilePath = Config File Path
+artifactory.task.collectIssues.configJson = Config Json
```

Comment:
I suggest changing to:
**artifactory.task.collectIssues.config.source = Config Source**
to be compatible with "Spec Source".